share/spice/phlbikeshare/phlbikeshare.js

```diff
@@ -0,0 +1,30 @@
+(function (env) {
```

Comment:
None of those should be in there. I thought I deleted them. I will check it
out. Thanks.
On Sat, Aug 15, 2015 at 3:41 PM Mariano Simone <notifications@github.com>
wrote:

> In share/spice/phlbikeshare/phlbikeshare.js
> <https://github.com/duckduckgo/zeroclickinfo-spice/pull/2102#discussion_r37138604>
> :
>
> > @@ -0,0 +1,30 @@
> > +(function (env) {
>
> another one to remove
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/duckduckgo/zeroclickinfo-spice/pull/2102/files#r37138604>
> .
>